azure-self-hosted-runners/post-deployment-script.ps1

```diff
@@ -14,8 +14,7 @@ param (
     [string]$GithubActionsRunnerName,
 
     [Parameter(Mandatory = $false, HelpMessage = "Stop Service immediately (useful for spinning up runners preemptively)")]
-    [ValidateSet('true', 'false')]
-    [string]$StopService = 'true',
+    [bool]$StopService = $true,
```

Comment:
I had actually tried this, and that did _not_ get picked up because we pass a string from the GitHub workflow. So I would suggest to drop this change.

Reply:
Oh, wait... That was not the `post-deployment` script variable, but the workflow input, and there a `boolean` really would not work.

Reply:
Ok I understand, I'll change it to string again. Would a switch work instead? Then you just need to write -Stopservice and it's true so if you don't include the -Stopservice in your command it's false.
I'm kind of alergic against strings when you use true or fasle :P But I'm changing it.